xcube/core/gen/iproc.py

```diff
@@ -328,7 +328,9 @@ def process(self,
                                       geo_coding=geo_coding,
                                       output_geom=output_geom,
                                       is_y_reversed=True)
-
+            if output_geom.is_tiled:
+                if dataset.chunks.get('lat')[0] < dataset.chunks.get('lat')[-1]:
```

Comment:
```suggestion
                # The following condition may become true, if we have used rectified_dataset(input, ..., is_y_reverse=True)
                # In this case y-chunksizes will also be reversed. So that the first chunk is smaller than any other.
                # Zarr will reject such datasets, when written.
                if dataset.chunks.get('lat')[0] < dataset.chunks.get('lat')[-1]:
```